src/bakery/Interface.cpp

```diff
@@ -53,10 +53,11 @@ void Interface::servicePlanner() {
     cout << "[1]: Single van visiting all clients without delivery times or capacity" << endl;
     cout << "[2]: Single van visiting all clients with delivery times but no capacity" << endl;
     cout << "[3]: Multiple vans with client allocation, capacity and delivery times" << endl;
+    cout << "[4]: SCC component of the graph containing the Bakery" << endl;
```

Comment:
It's fine if this option exists but we still need to adapt the SCC algorithm used to the graph (directed/undirected) and apply the right one in the algorithms. We should ask the user if the graph is directed or not, and maybe put a flag in the input file too

Reply:
Also, we should probably show it with GraphViewer instead (another PR I guess)